datalake/src/main/java/com/sequenceiq/datalake/cm/ClouderaManagerRangerUtil.java

```diff
@@ -118,17 +109,61 @@ public boolean isCloudIdMappingSupported(String stackCrn) throws ApiException {
         return isCloudIdMappingSupported(rolesResourceApi, clusterName, rangerUserSyncRoleName);
     }
 
-    public Optional<ApiCommand> setAzureCloudIdentityMapping(String stackCrn, Map<String, String> azureUserMapping) throws ApiException {
+    private Map<String, String> userMappingStrToMap(String userMappingStr) {
+        userMappingStr = userMappingStr.trim();
+        if (userMappingStr.isEmpty()) {
+            return Collections.emptyMap();
```

Comment:
I don't prefer returns in the middle of a method. Could you add an else branch for the rest f the code, so it's obvious there is 2 exit point?

Reply:
shall we log here it's empty?

Reply:
Fixed first issue, added log statement as suggested.